Libraries/Alert/NativeAlertManager.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2004-present Facebook. All Rights Reserved.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+
+type Button = {|
```

Comment:
Should we reuse these styles inside Alert.js instead of having them separately? Something like is done for [DialogOptions](https://github.com/facebook/react-native/blob/2b5b22f02fc5df253ea63c99d02c0adde028956a/Libraries/Alert/Alert.js#L16)